concent_api/command_line_tool/message_handler.py

```diff
@@ -0,0 +1,105 @@
+import os
+import requests
+from golem_messages.shortcuts import dump
+from golem_messages.shortcuts import load
+from golem_messages.message.concents import ForceGetTaskResult
+from golem_messages.message.concents import ForceReportComputedTask
+from golem_messages.message.concents import ForceSubtaskResults
+from golem_messages.message import AckReportComputedTask
+
+from api_testing_common import print_golem_message
+from api_testing_common import create_client_auth_message
+from utils.helpers import sign_message
+from utils.helpers import get_field_from_message
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "concent_api.settings")
+
+KEY_MAP = {
+    'requestor': [ForceGetTaskResult, AckReportComputedTask],
+    'provider': [ForceReportComputedTask, ForceSubtaskResults],
+}
+
+
+def print_message(message, cluster_url, endpoint):
+    if endpoint == 'send':
+        message_info = 'Endpoint: SEND'
+        _print_message_info(message_info)
+        message_info = ('Message: ' + str((type(message).__name__)) + ' SENT on: ' + str(cluster_url))
+        _print_message_info(message_info)
+    elif endpoint in ('receive', 'receive-out-of-band'):
+        message_info = f'Endpoint: {str.upper(endpoint)}'
+        _print_message_info(message_info)
+    elif message != None:
+        message_info = ('Response: ' + str((type(message).__name__)))
+        _print_message_info(message_info)
+    else:
+        print('Unsupported Message')
+    if message != None:
+        print_golem_message(message, indent=4)
+
+
+def _print_message_info(message_info):
+    print('-' * len(message_info) + '\n' + str(message_info) + '\n' + '-' * len(message_info))
+
+
+class MessageHandler():
+    def __init__(
+            self,
+            requestor_private_key,
+            requestor_public_key,
+            provider_public_key,
+            provider_private_key,
+            concent_pub_key):
```

Comment:
Same here with `(`, also there is to much indent.

Reply:
Done.